mypy.ini

```diff
@@ -1,5 +1,44 @@
 [mypy]
-ignore_missing_imports = True
 
 [mypy-API.migrations.*]
+# Auto-generated code
 ignore_errors = True
+
+# External packages that we haven’t set up types for
+
+[mypy-Levenshtein.*]
+ignore_missing_imports = True
+
+[mypy-colorama.*]
+ignore_missing_imports = True
+
+[mypy-cree_sro_syllabics.*]
+ignore_missing_imports = True
+
+[mypy-dawg.*]
+ignore_missing_imports = True
+
+[mypy-debug_toolbar.*]
+ignore_missing_imports = True
+
+[mypy-django.*]
+# TODO: try https://pypi.org/project/django-stubs/
```

Comment:
Could you try it in this PR?

Reply:
If you have more patience that I do, here’s a start: https://gist.github.com/andrewdotn/902b8062bf35bd3a3eb1cbb530500392

  - The `NewType`-casts, e.g., `FSTTag(foo)` do nothing at runtime if passed a completely different type
  - Not sure how to annotate a model `CharField` as yielding a `NewType()`-created class
  - Building up a `TypedDict` starting from `{}` really upsets mypy, and I would prefer not to have to drag in whatever is the mypy equivalent of TypeScript’s `Partial<>`